libs/features/components/ui-lists/src/lib/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.scss

```diff
@@ -0,0 +1,19 @@
+.center{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+ul{
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+li{
+    list-style-type: none;
+    height: 20px;
+    width: 30px;
+    padding: 30px;
+}
+
```

Comment:
Not styles for mobile view, should be a vertical list on mobile with org logos in pairs of 2 in each row

Reply:
@IanOdhiambo9 I've Implemented styles for the mobile view .